src/components/FVApps/FVAppsData.js

```diff
@@ -1,17 +1,15 @@
 import { useSites } from 'common/dataHooks/useSites'
 
 function FVAppsData() {
-  const sites = useSites()
+  // Passing a number to useSites will set the page size to that number
+  const sites = useSites(250)
```

Comment:
I attempted to set this number dynamically using the "count" value in the api response, but it would require a 2nd api query. I'm not sure if it's worth it to run 2 api queries each time this page is loaded rather than setting the pagesize manually

Reply:
Agreed, not worth a 2nd api query for this case.